### ISSUE TITLE
Getting invalid character 

### ISSUE THREAD

**Issue by hamidniaeml** [dbeaver/dbeaver#7767]
This query returns result in sqldeveloper but in dbeaver getting invalid character error
```
SELECT table_name,
CASE
  WHEN owner='SYS' THEN 'The owner is SYS'
  WHEN owner='SYSTEM' THEN 'The owner is SYSTEM'
  ELSE 'The owner is another value'
END
FROM all_tables;
```
log 
```
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [911] [22019]: ORA-00911: invalid character


	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:134)

	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:486)

	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$0(SQLQueryJob.java:424)

	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:159)

	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:416)

	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:776)

	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:2861)

	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:110)

	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:159)

	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:108)

	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$17.run(ResultSetViewer.java:3454)

	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:103)

	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

Caused by: java.sql.SQLSyntaxErrorException: ORA-00911: invalid character


	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)

	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1059)

	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:522)

	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:257)

	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:587)

	at oracle.jdbc.driver.T4CStatement.doOall8(T4CStatement.java:210)

	at oracle.jdbc.driver.T4CStatement.doOall8(T4CStatement.java:30)

	at oracle.jdbc.driver.T4CStatement.executeForDescribe(T4CStatement.java:762)

	at oracle.jdbc.driver.OracleStatement.executeMaybeDescribe(OracleStatement.java:925)

	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1111)

	at oracle.jdbc.driver.OracleStatement.doScrollExecuteCommon(OracleStatement.java:5179)

	at oracle.jdbc.driver.OracleStatement.doScrollStmtExecuteQuery(OracleStatement.java:5302)

	at oracle.jdbc.driver.OracleStatement.executeInternal(OracleStatement.java:1796)

	at oracle.jdbc.driver.OracleStatement.execute(OracleStatement.java:1745)

	at oracle.jdbc.driver.OracleStatementWrapper.execute(OracleStatementWrapper.java:334)

	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:338)

	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)

	... 12 more


```

**Comment by j-coates** [comment]
If you highlight the query and click execute, it'll run, so it's not the query itself. I've had it on some queries and not on others,

**Comment by uslss** [comment]
it seems this issue is duplicate of #7701 .
will be fixed in 6.3.4. Also fix is available in EA version (https://dbeaver.io/files/ea/).
Anyway, thanks for the bug report
